src/actions/rotate/IRotateAction.ts

```diff
@@ -0,0 +1,7 @@
+import {IAction} from "../../interfaces/IAction";
+
+export interface IRotateAction extends IAction{
+  angle(angle: number): IRotateAction;
```

Comment:
Are these functions the interface of Action? (Do all Actions implement these functions?)

We probably need an IRotateAction interface